examples/spider_example.py

```diff
@@ -0,0 +1,36 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from buildtest.modules.spider import Spider
+from buildtest.modules.util import Module
+
```

Comment:
An organizational question - why is Module imported from util? It would be more intuitive to have like:

```
from buildtest.modules import Module
```
But you don't want it accessible via the init?

Reply:
this makes sense to me if the file is `buildtest/modules.py` i can do that. I was not sure if i will be implementing other class so for instance `buildtest/modules/spider.py` was added to avoid having one massive file with all classes and methods.

I can make the file change if it makes sense.

Reply:
But you don't want to write it into an __init__.py in the modules folder?